src/views/Settings.vue

```diff
@@ -303,11 +305,15 @@ export default {
     return {
       edgeAddress: undefined,
```

Comment:
@vickywane this variable name confuses me each time I review this code. Let's rename it to something more self explanatory. Maybe `manualEdgeAddress` so it is clear this is only used when the user enters manually a remote edge address value and it has nothing to do with the currently connected edge address or the auto discovery mode edge address.

Reply:
Alright. 

I have modified this state and updated other references